libraries/aws/provisioning/src/aws_iot_provisioning_parser.c

```diff
@@ -120,88 +120,91 @@ static AwsIotProvisioningError_t _parseRejectedResponse( IotSerializerDecoderObj
     AwsIotProvisioning_Assert( pResponseData != NULL );
     AwsIotProvisioning_Assert( pStatusCode != NULL );
 
+    IotSerializerError_t decoderResult = IOT_SERIALIZER_SUCCESS;
     IotSerializerDecoderObject_t statusCodeDecoder = IOT_SERIALIZER_DECODER_OBJECT_INITIALIZER;
     IotSerializerDecoderObject_t errorCodeDecoder = IOT_SERIALIZER_DECODER_OBJECT_INITIALIZER;
     IotSerializerDecoderObject_t errorMessageDecoder = IOT_SERIALIZER_DECODER_OBJECT_INITIALIZER;
 
     /* Suppress warning when parameter is unused in non-Debug mode. */
     ( void ) pOperationName;
 
-    /* Initialize the status as "server refused" as we are parsing the "rejected" response. */
-    IOT_FUNCTION_ENTRY( AwsIotProvisioningError_t, AWS_IOT_PROVISIONING_SERVER_REFUSED );
+    AwsIotProvisioningError_t status = AWS_IOT_PROVISIONING_SUCCESS;
```

Comment:
What does it mean when a function named _parseRejectedResponse() returns provisioning success?  Does that mean parse success?

Reply:
Yes, as the goal of the function is to only parse the payload, my thought is to not add more responsibility/information in its return value than what its purpose warranties :)

Reply:
OK.  Maybe in the future we can make these constants less confusing.